meshtastic/mqtt.proto

```diff
@@ -30,4 +31,71 @@ message ServiceEnvelope {
    * the globally trusted nodenum
    */
   string gateway_id = 3;
+}
+
+/*
+ * Information about a node intended to be reported unencrypted to a map using MQTT.
+ */
+message MapReport {
```

Comment:
Perhaps firmware version would be another good data point?

Reply:
Good one! Will add it. We don't really need to worry about size since it's MQTT only anyway.